docs/integrations/bridges/cbridge.md

```diff
@@ -61,6 +61,16 @@ After the transaction, you can see
 
 and check out your Metamask!
 
+## Understanding the difference betweeen USDT vs. cUSDT
+
+Recently, Tether announced the launch of USDT on Kusama (a scalable network of specialized blockchains built using Substrate and nearly the same codebase as Polkadot), which is the native USDT token on the Kusama blockchain.
```

Comment:
Tether is already live on Kusama for some time (https://tether.to/en/tether-token-usdt-launches-on-kusama/). The recent announcement was that it would also come to Polkadot.